logback-classic/src/test/java/ch/qos/logback/classic/spi/PackagingDataCalculatorTest.java

```diff
@@ -119,4 +120,12 @@ public void noClassDefFoundError_LBCLASSIC_125Test() throws MalformedURLExceptio
 
     }
 
+    @Test
+    // Test http://jira.qos.ch/browse/LOGBACK-1146
+    public void LOGBACK_1146Test() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/tony19/logback/file/2371174609/issues/source?bid=3222618&fileBranchId=3222660#l125)